examples/aws/module_with_specific_provider/providers.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.11.0"
+    }
+  }
+
+  required_version = "1.5.5"
+}
+
+// default provider that is used when no other provider
+// is specified explicitly
+provider "aws" {
+  region = "eu-west-1"
+  alias  = "specific"
+
+  default_tags {
+    tags = {
+      Project   = "they-terraform-examples"
+      CreatedBy = "terraform"
+    }
+  }
+}
```

Comment:
just wondering, we already have something similar in the example for api_gateway_with_domain where we set different providers for different resources, this isn't any different is it? 
I still think it's good and useful to have an explicit example, but I want to make sure I've understood both setups :D

Reply:
It is basically the same, just that within the api_gateway example, we use a different provider for the `data "aws_acm_certificate" "acm_certificate"` data resource that comes from terraform. In this new example, we set a specific provider for one of our modules (and show that it works the same)

Reply:
Thx for elaborating on this! 💕